testing/integration/upgrade_uninstall_test.go

```diff
@@ -80,6 +81,18 @@ func TestStandaloneUpgradeUninstallKillWatcher(t *testing.T) {
 	}
 	require.NoError(t, err)
 
+	// on windows watcher needs to start before uninstall,
+	// otherwise you can get a race condition where watcher hasn't
+	// started yet, so it doesn't show up in PID list, but will
+	// start before deletion of directory that will lead to
+	// sharing violation on Windows
```

Comment:
Why is this only true on Linux? Won't we also fail to find the watcher PID there?

It won't cause a sharing violation, but it will let the watcher keep running and potentially interfere with the next test, which is the reason why we started killing it.

Reply:
I think we should get rid of the GOOS restriction here.

Reply:
switched to all OSes